client/src/Components/Reservations/reducer.js

```diff
@@ -19,6 +21,18 @@ const reservationsReducer = (state = initialState, action) => {
         loading: false,
         reservations: action.reservations
       };
+    case actions.PROPERTIES_SUCCESS:
+      return {
+        ...state,
+        loading: false,
+        properties: action.properties
```

Comment:
You use the payload variable name to keep consistency here. You don't have to remember what you named the action's payload.
You can also destructure the action parameter if you do that.
```
const reducer = (state = initialState, {type, payload}) => {...}
```